readme.md

```diff
@@ -76,7 +78,7 @@ gulp.task('jade', function () {
 
 ##### hasChanged
 
-Type: `function`  
+Type: `function`
```

Comment:
Add two spaces at the end of the line so that `Default...` starts on a new line.